pkg/build/api/types.go

```diff
@@ -82,6 +82,9 @@ type BuildStatus struct {
 	// Duration contains time.Duration object describing build time.
 	Duration time.Duration
 
+	// Pushspec contains a resolved Docker pushspec computed from Build.Spec.Output.To.
+	Pushspec string
```

Comment:
also we're considering supporting multiple outputs in the future, maybe we should future proof and make this an array...what do you think @smarterclayton ?